.isort.cfg

```diff
@@ -0,0 +1,2 @@
+[settings]
+profile=black
```

Comment:
Might possibly want end-of-file-fixer to do cfgs too?
https://github.com/pre-commit/pre-commit-hooks/issues/125#issuecomment-228810449